server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -67,6 +67,7 @@
 
     public static final String CRON_GROUP = "cron group";
     public static final String SINGLE_JOB_GROUP = "async group";
+    public static final String[] DELETED_JOBS = new String[] { "StatisticHistoryTask" };
```

Comment:
@mstead after discussion with @wottop earlier today, im concerned about adding a config option.
In addition to the above reasons, essentially users of the application will be able to delete jobs that are mentioned in that config whether or not that job was deleted.... TL-DR, that option could be abused as well..